vm/src/stdlib/time.rs

```diff
@@ -102,6 +102,11 @@ mod time {
         Ok(Date::now() / 1000.0)
     }
 
+    #[cfg(all(target_arch = "wasm32", not(feature = "js"), not(target_os = "wasi")))]
+    fn _time(vm: &VirtualMachine) -> PyResult<f64> {
+        Err(vm.new_not_implemented_error("time.time".to_owned()))
```

Comment:
I will need a way to implement time, I have the APIs in my environment, but I will need a way to override this